src/Arcus.BackgroundJobs.AzureActiveDirectory/ClientSecretExpirationJob.cs

```diff
@@ -89,14 +89,14 @@ public async Task ExecuteAsync(CancellationToken stoppingToken)
                 if (application.RemainingValidDays < 0)
                 {
                     eventType = ClientSecretExpirationEventType.ClientSecretExpired;
-                    _logger.LogEvent($"The secret {application.KeyId} for Azure Active Directory application {application.Name} has expired.", telemetryContext);
+                    _logger.LogSecurityEvent($"The secret {application.KeyId} for Azure Active Directory application {application.Name} has expired", telemetryContext);
```

Comment:
I remember that I already asked this in the past, but can't find it anymore.
I just wonder what the added value is of having a `LogSecurityEvent` extension method.  Aren't we going to fine grained in this ?

Reply:
It's mostly to be able to easier filter out these kind of critical events in logging systems. We use the `LogSecurityEvent`, for example is all security-related components in Arcus (secret store, web API authentication filters, key rotation...). This built-in functionality makes the telemetry easier to filter, mostly. Seems like a good practice to create a by-default split between security events and other stuff.

Reply:
I can live with that, but to be honest, I do not see much added value here.